guardian/open-journalism
# Core Web Vitals scoring

## What does this change?

Capture the function from Lighthouse to calculate log-normal scores.

## How to test 

```SQL
CREATE TEMP FUNCTION getLogNormalScore(value NUMERIC, metric STRING)
RETURNS NUMERIC
LANGUAGE js
AS r"""
/* make sure the JavaScript is imported before here */
return getScore(metric, value);
""";
```
